composer.json

```diff
@@ -12,14 +12,20 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.4",
```

Comment:
I like that you added `ext-gd`, updated the PHPUnit version to the one we should be using, and added `autoload-dev`.
Yet, as per [this statistic](https://w3techs.com/technologies/details/pl-php/5/all), about 14% of ALL WEBSITES with a known server-side language are still on PHP 5.3, which is why I would like to avoid dropping it.